sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -531,6 +531,7 @@ public partial class HttpPipeline
         public HttpPipeline(Azure.Core.Pipeline.HttpPipelineTransport transport, Azure.Core.Pipeline.HttpPipelinePolicy[]? policies = null, Azure.Core.ResponseClassifier? responseClassifier = null) { }
         public Azure.Core.ResponseClassifier ResponseClassifier { get { throw null; } }
         public static System.IDisposable CreateClientRequestIdScope(string? clientRequestId) { throw null; }
+        public static System.IDisposable CreateHttpMessagePropertyScope(string name, object value) { throw null; }
```

Comment:
Maybe worth changing it to take a dictionary so we don't have to create multiple scopes to set multiple properties.